lib/Common/PlatformAgnostic/Exception.h

```diff
@@ -0,0 +1,15 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#ifdef __clang__
```

Comment:
BTW, wondering if I misunderstand `__clang__`. Why use this compiler symbol to determine platform feature? Does clang on Windows support SEH?